regression1.Rmd

```diff
@@ -747,7 +745,6 @@ sacr_multi <- workflow() |>
 
 sacr_k <- sacr_multi |>
               pull(neighbors)
-
```

Comment:
Not sure I know why spaces are being deleted here -- any particular reason?

Reply:
For spacing of the code blocks spilling onto new pages

Reply:
nope, just wasn't sure why the deletion happened! if it looks good to you, then i'm happy with it